core-services/prow/02_config/_boskos.yaml

```diff
@@ -1,15 +1,378 @@
+  - - us-west-1--08
+  - - us-west-1--09
+  - - us-west-1--10
+  - - us-west-1--11
+  - - us-west-1--12
+  - - us-west-1--13
+  - - us-west-1--14
+  - - us-west-1--15
+  - - us-west-1--16
+  - - us-west-1--17
+  - - us-west-1--18
+  - - us-west-1--19
+  - - us-west-1--20
+  - - us-west-1--21
+  - - us-west-1--22
+  - - us-west-1--23
+  - - us-west-1--24
+  - - us-west-1--25
+  - - us-west-1--26
+  - - us-west-1--27
+  - - us-west-1--28
+  - - us-west-1--29
+  - - us-west-1--30
+  - - us-west-1--31
+  - - us-west-1--32
+  - - us-west-1--33
+  - - us-west-1--34
+  - - us-west-1--35
+  - - us-west-1--36
+  - - us-west-1--37
+  - - us-west-1--38
+  - - us-west-1--39
+  - - us-west-1--40
+  - - us-west-1--41
+  - - us-west-1--42
+  - - us-west-1--43
+  - - us-west-1--44
+  - - us-west-1--45
+  - - us-west-1--46
+  - - us-west-1--47
+  - - us-west-1--48
+  - - us-west-1--49
+  - - us-west-2--00
+  - - us-west-2--01
+  - - us-west-2--02
+  - - us-west-2--03
+  - - us-west-2--04
+  - - us-west-2--05
+  - - us-west-2--06
+  - - us-west-2--07
+  - - us-west-2--08
+  - - us-west-2--09
+  - - us-west-2--10
+  - - us-west-2--11
+  - - us-west-2--12
+  - - us-west-2--13
+  - - us-west-2--14
+  - - us-west-2--15
+  - - us-west-2--16
+  - - us-west-2--17
+  - - us-west-2--18
+  - - us-west-2--19
+  - - us-west-2--20
+  - - us-west-2--21
+  - - us-west-2--22
+  - - us-west-2--23
+  - - us-west-2--24
+  - - us-west-2--25
+  - - us-west-2--26
+  - - us-west-2--27
+  - - us-west-2--28
+  - - us-west-2--29
+  - - us-west-2--30
+  - - us-west-2--31
+  - - us-west-2--32
+  - - us-west-2--33
+  - - us-west-2--34
+  - - us-west-2--35
+  - - us-west-2--36
+  - - us-west-2--37
+  - - us-west-2--38
+  - - us-west-2--39
+  - - us-west-2--40
+  - - us-west-2--41
+  - - us-west-2--42
+  - - us-west-2--43
+  - - us-west-2--44
+  - - us-west-2--45
+  - - us-west-2--46
+  - - us-west-2--47
+  - - us-west-2--48
+  - - us-west-2--49
   state: free
   type: aws-quota-slice
-- max-count: 30
-  min-count: 30
+- names:
+  - - centralus--00
+  - - centralus--01
+  - - centralus--02
+  - - centralus--03
+  - - centralus--04
+  - - centralus--05
+  - - centralus--06
+  - - centralus--07
+  - - centralus--08
+  - - centralus--09
+  - - centralus--10
+  - - centralus--11
+  - - centralus--12
+  - - centralus--13
+  - - centralus--14
+  - - centralus--15
+  - - eastus1--0
+  - - eastus1--1
+  - - eastus1--2
+  - - eastus1--3
+  - - eastus1--4
+  - - eastus1--5
+  - - eastus1--6
+  - - eastus1--7
+  - - eastus1--8
+  - - eastus1--9
+  - - eastus2--0
+  - - eastus2--1
+  - - eastus2--2
+  - - eastus2--3
+  - - eastus2--4
+  - - eastus2--5
+  - - eastus2--6
+  - - eastus2--7
+  - - eastus2--8
+  - - eastus2--9
+  - - westus--0
+  - - westus--1
+  - - westus--2
+  - - westus--3
+  - - westus--4
+  - - westus--5
+  - - westus--6
+  - - westus--7
+  - - westus--8
+  - - westus--9
   state: free
   type: azure4-quota-slice
-- max-count: 120
-  min-count: 120
+- names:
+  - - us-east1--000
```

Comment:
Nit: Experience teaches that if you gurantee a certain number of digits, ppl will assume this is always true: https://www.hyrumslaw.com/ 

Hence, its usually better to just count up. WDYT?

Reply:
openshift/ci-tools#1306 hides this from consumers, and with 0ec8261ff8 I have a guard here to keep folks from accidentally tripping over the ci-tools logic.